webapp/src/main/java/com/box/l10n/mojito/rest/openai/OpenAIWS.java

```diff
@@ -0,0 +1,99 @@
+package com.box.l10n.mojito.rest.openai;
+
+import com.box.l10n.mojito.entity.AIPrompt;
+import com.box.l10n.mojito.service.openai.OpenAIService;
+import io.micrometer.core.annotation.Timed;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ConditionalOnProperty(value = "l10n.openai.enabled", havingValue = "true")
+public class OpenAIWS {
```

Comment:
To discuss: I wonder if this class (WS) & the underlying service should be split into two separate classes/services with different responsibilities:
1. One to deal with prompt management in general "api/aiprompts/..." to create, delete & retrieve specific prompts
2. One to deal with the specific LLM execution/applying the prompts on content (LLMService/OpenAIService/ClaudeService etc.) - similar to our third party abstraction

Reply:
Agree on both here. I'll update the PR to have on LLMService interface which is implemented by the relevant service classes. 

A type config param will be introduced to choose which LLM service is in use (OpenAI only option for now)